core/arch/arm/plat-rockchip/platform_config.h

```diff
@@ -85,6 +85,25 @@
 #define FIREWALL_DDR_BASE	0xff534000
 #define FIREWALL_DDR_SIZE	SIZE_K(16)
 
+#elif defined(PLATFORM_FLAVOR_rk3588)
```

Comment:
Having a SoC as the name of the flavor is not a good idea IMO. How would you name different boards using the same SoC? Although yes, ideally if we use DT and runtime detection etc. it may be possible but it doesn't seem to apply here.